myrails/configuration/Readme.md

```diff
@@ -0,0 +1,46 @@
+# MySQL Database Setup
```

Comment:
one issue I faced when installing MySQL on my machine, getting errors on installing the requirement `mysqlclient==2.0.1`
![Screenshot 2020-11-10 013447](https://user-images.githubusercontent.com/26089554/98608991-0c158180-22f5-11eb-8993-fbbb58b1a40d.png)

Reply:
@murradkh  Not sure what is going on, try **[these steps](https://stackoverflow.com/questions/47870628/oserror-mysql-config-not-found-when-trying-to-pip-install-mysqlclient-dja)** and let me know if you are still having the same issue